pyphare/pyphare/pharesee/hierarchy/patch.py

```
@@ -67,3 +67,41 @@ def __call__(self, qty, **kwargs):
+        return type(self) is type(that) and self.patch_datas == that.patch_datas
+
+
+def patch_array_ufunc(patch, ufunc, method, *inputs, **kwargs):
+    if method != "__call__":
+        return NotImplemented
+
+    if not all([patch.is_compatible(o) for o in inputs]):
+        raise TypeError("Patch: incompatible arguments")
+
+    pds = [x.patch_datas.values() for x in inputs]
+    out = [getattr(ufunc, method)(*pd, **kwargs) for pd in zip(*pds)]
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)